### DISCUSSION TITLE
It is possible to integrate both, com.rest.elevenlabs with com.rest.openai? (Unity)

### DISCUSSION THREAD

**Discussion by DakkuaDev** [RageAgainstThePixel/com.rest.elevenlabs#84] [Q&A]
Hello! I am making a chatbot client with Unity Engine that uses Microphone native Unity class to record user's voice. Then I send the transcribed text to OpenAI chat endpoint with your library (com.rest.openai). My next step is to convert text-to-voice using Whisper. But because all voices are optimised for English, it sounds weird in Spanish. I used ElevenLab in the past, and it comes with Spanish cloned voices that sound great. Can I integrate seamlessly between these two?

**Comment by StephenHodgson** [comment]
Yes, I have a premium package for purchase that combines both of these packages into one.

DM me on discord for more details.
